nomad/structs/variables.go

```diff
@@ -505,3 +592,37 @@ type VariablesReadResponse struct {
 	Data *VariableDecrypted
 	QueryMeta
 }
+
+// VariablesRenewLockRequest is used to renew the lease on a lock. This request
+// behaves like a write because the renewal needs to be forwarded to the leader
+// where the timers and lock work is kept.
+type VariablesRenewLockRequest struct {
+	Namespace string
```

Comment:
The embedded `WriteRequest` struct already has the `Namespace` field, so I don't think we need it here.

Reply:
Indeed, removed